src/Commands/CreatePdf.php

```diff
@@ -0,0 +1,95 @@
+<?php
+
+namespace Drupal\os2forms_digital_post\Commands;
+
+use Drupal\Core\Entity\EntityTypeManagerInterface;
+use Drush\Commands\DrushCommands;
+use Drupal\os2forms_digital_post\Manager\TemplateManager;
+use Drupal\Core\Config\ConfigFactoryInterface;
+
+/**
+ * A drush command file.
+ *
+ * @package Drupal\event_database_pull\Commands
+ */
+class CreatePdf extends DrushCommands {
+
+  /**
+   * The os2forms_digital_post template manager.
+   *
+   * @var TemplateManager
+   */
+  protected TemplateManager $templateManager;
+
+  /**
+   * The entity type manager.
+   *
+   * @var \Drupal\Core\Entity\EntityTypeManagerInterface
+   */
+  protected $entityTypeManager;
+
+  public function __construct(TemplateManager $templateManager, EntityTypeManagerInterface $entity_type_manager) {
+    parent::__construct();
+    $this->templateManager = $templateManager;
+    $this->entityTypeManager = $entity_type_manager;
+  }
+
+  /**
+   * Create PDF named test.pdf in current directory.
+   *
+   * @param string $template
+   *   The template name to use.
+   * @param array $options
+   *   An option that takes multiple values.
+   *
+   * @command os2forms_digital_post:create_pdf
+   * @aliases create-pdf
+   * @usage os2forms_digital_post:create_pdf default --submission_id=12345
+   *   Create PDF using default template with data from set submission.
+   *
+   * @throws \Drupal\Component\Plugin\Exception\InvalidPluginDefinitionException
+   * @throws \Drupal\Component\Plugin\Exception\PluginNotFoundException
+   */
+  public function create($template, $options = ['submission_id'=> 0]) {
+    $elements[] = [];
+    $webformLabel = '';
+    if (0 < $options['submission_id']) {
```

Comment:
This check seems superfluous. You could check that `$options['submission_id']` is actually an integer, but just skipping any checks would be fine, i.e. the code will work and report any invalid submission id.

Reply:
REmoved